src/Query.php

```diff
@@ -27,10 +31,14 @@ public function build() {
 
 	protected function buildQuery() {
 		$signature = $this->renderSignature();
-		return "query $signature{\n" . $this->render(1) . "}\n\n";
+		return "$this->type $signature{\n".$this->render(1)."}\n\n";
+	}
+
+	public function setType($type) {
+		$this->type = $type;
 	}
 
-	protected function renderSignature() {
+    protected function renderSignature() {
```

Comment:
Leading white space again. I think `git diff` would tell you this. Or your editor could.

Reply:
fixed